okio/src/jsMain/kotlin/okio/FileSource.kt

```diff
@@ -33,11 +34,13 @@ internal class FileSource(
       buffer = data,
       length = byteCount,
       offset = 0,
-      position = null
+      position = position_.toDouble()
```

Comment:
Why double here?

Reply:
In JS it's declared to accept `Number` but I found it didn't work properly until I manually converted to JS’ only native number type.